resources/assets/js/mix/trainingticket.js

```diff
@@ -44,41 +44,48 @@ if (newForm.length || (editForm.length && draft.length)) {
   }, 60000);
 }
 
-var stars = $("#stars span");
+const stars = document.querySelectorAll("#stars span");
 
 // Needed for if you submit and get redirected back the stars will now still be highlighted
-var rating = $("#stars input[name='score']").val();
+var rating = document.querySelector("#stars input[name='score']").value;
 if (rating) {
   for (var i = 0; i < rating; i++) {
```

Comment:
`rating` could be a const, `i` could be a let

Reply:
Done